winlogbeat/eventlog/eventlog.go

```diff
@@ -139,7 +138,7 @@ func (e Record) ToEvent() beat.Event {
 // MapStr.
```

Comment:
A few lines up...

 `e.TimeCreated.SystemTime` is used to populate `@timestamp`. That's the time the event was originally logged. So it would be nice to populate `event.created` with the current time.

Reply:
Done